### DISCUSSION TITLE
feat: Edit metadata #5066

### DISCUSSION THREAD

**Discussion by bearded-papa** [immich-app/immich#5499] [Q&A] [answered]
I have just read the above and wondered if this was a feature that is already implemented as I noticed in the video that the server version was a previous one to the one I am running.
The reason I as is I have looked for this feature on my server and am unable to find it.

**Comment by bo0tzz** [comment, marked as answer]
The PR was merged after the latest release, so it'll be in 1.90.0

**Reply by bearded-papa** [reply]
Thanks for clarification
